CMakeLists.txt

```diff
@@ -89,6 +91,13 @@ include_directories(
         ${WAMR_IWASM_DIR}/common
 )
 
+#WAMR Interpreter / AoT Switch
+if(WAMR_EXECUTION_MODE STREQUAL "Interpreter")
```

Comment:
With the interpreter mode as an option, it can change to:

```
if(WAMR_INTERPRETER_MODE)
...
```

This avoids the need to do string matching.